rkt/cli_apps.go

```diff
@@ -304,3 +304,151 @@ func (aml *appCPULimit) String() string {
 func (aml *appCPULimit) Type() string {
 	return "appCPULimit"
 }
+
+type appHashType apps.Apps
+
+func (aa *appHashType) IsBoolFlag() bool {
+	return true
+}
+
+func (aa *appHashType) Set(s string) error {
+	app := (*apps.Apps)(aa).Last()
+	if app == nil {
+		return fmt.Errorf("--is-hash must follow an image")
```

Comment:
Looks like I forgot to hook them up. :/ See commit https://github.com/kinvolk/rkt/commit/9717b263897e198345b8a2f15973d435987929ff

Anyway, I'll file this as a separate PR.